### PR TITLE
BF-25 Clear situation

### PR DESCRIPTION
**Pull request by var-kyle**
# Description

[BF-25 Clear my situation](https://bf-decd.atlassian.net/browse/BF-25?atlOrigin=eyJpIjoiOWUxNTYyMWJiOTMwNGU3ZGI2MGE0YzI3OTEyMGU3Y2EiLCJwIjoiaiJ9)

Add a button in the `My situation` section to clear the fields as well as telling the server to delete the cookie.

## Meets Definition of Done
- [ ] Unit tests up to date
- [ ] E2E tests up to date
- [ ] I18n

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -56,6 +56,15 @@ export default function Home({
     }));
   };
 
+  const clearSituation = () => {
+    fetch(`${process.env.NEXT_PUBLIC_BASE_URL}/api/situation`, {
+      method: "delete",
+    });
```
Comment (DaceyTom2) [inline pages/index.js:62]: The cookie gets recreated immediately after by setting situation so this call doesn't really do anything

File: pages/index.js [hunk 2]
```diff
@@ -56,6 +56,15 @@ export default function Home({
...
+    document.getElementById("location-select").value = "";
+    document.getElementById("age").value = "";
+    document.getElementById("income-select").value = "";
```
Comment (DaceyTom2) [inline pages/index.js:65]: It is probably worth setting situation in state for consistency instead

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 2 above)
Comment (var-kyle) [reply pages/index.js:63]: Just calling `setSituation` doesn't clear the fields, so these are required.

Comment (DaceyTom2) [reply pages/index.js:63]: That is weird. It should be able to update based off of state. Something to look into in the future perhaps

**Review by DaceyTom2 — APPROVED** [review]
Awesome!

**Pull request merged**